app/src/main/java/com/worldsnas/starplayer/api/WebServiceApi.kt

```diff
@@ -0,0 +1,9 @@
+package com.worldsnas.starplayer.api
+
+import retrofit2.http.GET
+
+interface WebServiceApi {
+
+    @GET("/api/request")
```

Comment:
chera hich parametri pas nemidim?
server bedone meghad chi bar migardone? (hame ye list ro?!!!!!)

Reply:
chon inja kolan list hame ahang hai ke harooz server migere miyare ro neshon mide
vali linke haye dge hast ke maslan byad search bezanim dakhele ahang on moghe parametre vorodi migre.
ke harche zood tar in feature o ezafe mikonam

Reply:
Lotfan ba paging piade konid

Reply:
> Lotfan ba paging piade konid

chashm